src/utils/mmtQuery.js

```diff
@@ -47,7 +47,7 @@ export const mmtQuery = ({
     data: cmrParameters,
     headers: permittedHeaders,
     method: 'GET',
-    url: `${process.env.mmtRootUrl}/api/drafts/${id}?draft_type=${draftType}`
+    url: `${process.env.mmtRootUrl}/api/${draftType}/${id}`
```

Comment:
It seems like it might have been more simple if we swapped this line out with ``url: `${process.env.mmtRootUrl}/api/${draftType}_drafts/${id}`` which might've avoided the need for the changes to `draftConcept.js`. Any reason it was not implemented that way?

Reply:
Updated the PR so now I'm passing draftType to mmtQuery.js as `collectionDraft` and then on line 50 I'm calling snakeCase and adding an 's'.